internal/app/machined/pkg/runtime/v1alpha1/v1alpha1_sequencer.go

```diff
@@ -473,21 +471,12 @@ func (*Sequencer) Upgrade(r runtime.Runtime, in *machineapi.UpgradeRequest) []ru
 			!r.Config().Machine().Kubelet().SkipNodeRegistration(),
 			"drain",
 			CordonAndDrainNode,
-		).AppendWhen(
-			!in.GetPreserve(),
-			"cleanup",
-			RemoveAllPods,
```

Comment:
same here, not needed?

Reply:
yes, same as above, no wiping on upgrade